packages/paste-website/src/pages/getting-started/design/index.mdx

```diff
@@ -42,136 +42,51 @@ export const pageQuery = graphql`
 
 <content>
 
-## Getting Started with Paste
+## Quickstart
 
-### Access for Font Files, Sketch Licenses, Abstract
+- Need to access for Font Files? [Instructions for Download](https://wiki.hq.twilio.com/x/sTQdCw) (Confluence)
 
-[Instructions for Download](https://wiki.hq.twilio.com/x/sTQdCw) (Confluence)
+- Paste uses Abstract as a version control system to serve design assets such as symbols and icons. If you’re unfamiliar with Abstract, version control, or git workflows, read through <a href="https://www.goabstract.com/how-it-works/">how it works</a>. After installing Abstract, sync the [Paste Components](https://share.goabstract.com/786b3d8f-b87a-4140-af7d-eb4cdd40e8ed?) project.
```

Comment:
No need to sync the project anymore, but PDs will need to link the component libraries and theme library to their project. I'm not at my computer right now, but I can open a PR to update this tomorrow if you want to merge tonight.